modules/mono/glue/cs_files/Mathf.cs

```diff
@@ -56,19 +62,20 @@ public static real_t Ceil(real_t s)
         {
             return (real_t)Math.Ceiling(s);
         }
+        
+        public static int CeilToInt(real_t s)
+        {
+            return (int)Math.Ceiling(s);
+        }
 
-        public static real_t Clamp(real_t val, real_t min, real_t max)
+        public static int Clamp(int value, int min, int max)
         {
-            if (val < min)
-            {
-                return min;
-            }
-            else if (val > max)
-            {
-                return max;
-            }
+            return value < min ? min : value > max ? max : value;
```

Comment:
Syntax highlighting treats `value` as a special word [because it's also a C# contextual keyword](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/value). Perhaps it's better to stick with `val`? @neikeq

Reply:
It have no drawbacks, as I mentioned before its valid as "from" keyword in Lerp/InverseLerp. Github highlight it as keyword when its used as parameter but its a Github bug.

Reply:
"value" is how its writed in GDScript and I just consist the syntaxes(the C# still in the beta right ?) and dont worry named parameters works as expected

Reply:
I'm fine with `value`.